astroquery/mast/core.py

```diff
@@ -32,7 +32,8 @@
 from ..query import QueryWithLogin
 from ..utils import commons, async_to_sync
 from ..utils.class_or_instance import class_or_instance
-from ..exceptions import TimeoutError, InvalidQueryError, NoResultsWarning, RemoteServiceError
+from ..exceptions import TimeoutError, InvalidQueryError, RemoteServiceError, LoginError
+from ..exceptions import NoResultsWarning
```

Comment:
no need to repeat the from part, simply list everything in parenthesis with a linebreak where needed after the first import